### DISCUSSION TITLE
More information in status bar

### DISCUSSION THREAD

**Discussion by aberenguel** [sepinf-inc/IPED#1108] [Ideas]
In the current version of IPED (4.0.0-snapshot), the status bar only shows the path of the last selected item.

It would be great if it showed other information like:
- total number of items in the table
- number of selected items
- sum size of selected items

Regarding the last selected item
- path
- hash
- length

**Comment by lfcnassif** [comment]
Hi @aberenguel,

This would be an useful enhancement.

But not sure if the status bar will have enough space to show all that info in medium sized screen.

**Reply by aberenguel** [reply]
The biggest part would be the path. I think it could be dedicated a proportion of screen width to it. 
If the path is huge, it can be truncated and suffixed with ellipsis. When the user holds the pointer on the path, it can be completely showed in a balloon. 
Just ideas for now...
